matter_server/common/custom_clusters.py

```diff
@@ -357,6 +357,94 @@ def attribute_type(cls) -> ClusterObjectFieldDescriptor:
             value: float32 = 0
 
 
+@dataclass
+class ThirdRealityMeteringCluster(Cluster, CustomClusterMixin):
+    """Custom (vendor-specific) PowerMetering cluster for ThirdReality."""
+
+    id: ClassVar[int] = 0x130DFC02
+
+    @ChipUtility.classproperty
+    def descriptor(cls) -> ClusterObjectDescriptor:
+        """Return descriptor for this cluster."""
+        return ClusterObjectDescriptor(
+            Fields=[
+                ClusterObjectFieldDescriptor(
+                    Label="currentSummationDelivered", Tag=0x0000, Type=uint
+                ),
+                ClusterObjectFieldDescriptor(
+                    Label="instantaneousDemand", Tag=0x0200, Type=uint
```

Comment:
Shouldn't this be 0x0400

```suggestion
                    Label="instantaneousDemand", Tag=0x0400, Type=uint
```

Reply:
oops